.github/config/labels.yml

```
@@ -0,0 +1,102 @@
+# Issue Types - Core labels for issue categorization
+- name: bug
+ color: d73a4a
+ description: Report an issue or unexpected behavior
+- name: feature
+ color: a2eeef
+ description: Request for new feature or enhancement
+ aliases:
+  - enhancement
+- name: question
+ color: d876e3
+ description: Request for information or clarification
```

Maybe add a line break between labels. I'm finding this pretty hard to ready atm with the density of it.
```suggestion
- name: bug
  color: d73a4a
  description: Report an issue or unexpected behavior
 
- name: feature
  color: a2eeef
  description: Request for new feature or enhancement
  aliases:
    - enhancement
  
- name: question
  color: d876e3
  description: Request for information or clarification
```

---

The current format follows the conventional YAML style guide where related metadata labels are kept together